docs/skopeo-copy.1.md

```diff
@@ -54,6 +54,10 @@ Directory to use to share blobs across OCI repositories.
 
 After copying the image, write the digest of the resulting image to the file.
 
+**--preserve-digests**
+
+Preserve the digests during copying. Fail if the digest cannot be preserved.
```

Comment:
Tentative suggestion, I’m not at all sure: `--preserve-digests` is a bit of typing. Maybe `--exact`?

OTOH users who do care about this almost certainly know about digests.

@vrothberg WDYT?

Reply:
I am very unhelpful since I like both :^)